src/typeahead/test/typeahead.spec.js

```diff
@@ -620,8 +636,6 @@ describe('typeahead tests', function () {
 
       // Note that this bug can only be found when element is in the document
       $document.find('body').append(element);
-      // Extra teardown for this spec
-      this.after(function () { element.remove(); });
```

Comment:
They need to be removed somehow, so perhaps setting it on a global variable and removing it on afterEach if the variable is set?